docs/administration/deployment/deploying-at-scale/index.md

```diff
@@ -13,7 +13,7 @@ REANA can be easily deployed on large Kubernetes clusters using Helm. Useful for
     For example, [CERN uses CephFS](https://clouddocs.web.cern.ch/containers/tutorials/cephfs.html) as the default storage backend.
 
 !!! note
-    If you do not have any particular distributed file system in your Kubernetes cluster, you can easily [deploy an NFS network file system following our documentation](../../advanced-usage/storage-backends/nfs).
+    If you do not have any particular distributed file system in your Kubernetes cluster, you can easily [deploy an NFS network file system following our documentation](../../../advanced-usage/storage-backends/nfs).
```

Comment:
Link does not work for me. I get 404 error after pressing it. Appending "/" or "/index.md" solves the problem

Reply:
Strange.. it works for me, maybe it's a browser problem? I'll append `/` just in case.